test/controllers/user.controller.test.js

```diff
@@ -1772,9 +1781,13 @@ describe('User Controller', () => {
               expect(res.body.success).to.be.eql(true);
               expect(res.body.message).to.be.eql('Vendor information updated');
               expect(res.body.user._id).to.be.eql(vendorId.toString());
+              expect(res.body.user.phone).to.be.eql(vendorUser.phone);
+              expect(res.body.user.phone2).to.be.eql(data.phone2);
+              expect(res.body.user.address.city).to.be.eql(vendorUser.address.city);
```

Comment:
Instead of testing for the address separate, you can combine the address and test them with one assertion.

```suggestion
              expect(res.body.user.address).to.be.eql({
                ...vendorUser.address,
                ...data.address,
              });
```